### ISSUE TITLE
Type declarations not found with "moduleResolution": "node"

### ISSUE THREAD

**Issue by johanrd** [hokulea/aria-voyager#171]
When importing the menu modifier in a project with tsconfig "moduleResolution": "node", I got the following error:

```
Cannot find module 'ember-aria-voyager' or its corresponding type declarations.
  There are types at '…/node_modules/ember-aria-voyager/declarations/index.d.ts', but this result could not be resolved under your current 'moduleResolution' setting. Consider updating to 'node16', 'nodenext', or 'bundler'.glint(2307)
```
 
Changing my module resolution to `nodenext` fixed this, but this has never been an issue with any other v2 addons I use, so I guess there is also something we can do about the exports here? (I may not be the only one). 

environment:
node v22.8.0
"@glint/core": "1.4.1-unstable.01b8deb",
